<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimonView</class>
 <widget class="QMainWindow" name="SimonView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimonView</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="playButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Terminal</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: #333;
    border: 2px solid #555;
    border-radius: 20px;
	background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #eee, stop: 1 #aaa
        );
}

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
        );
    }</string>
    </property>
    <property name="text">
     <string>Click to Play!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="leftRedButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>261</width>
      <height>271</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-image: url(&quot;:/icons/redFlashOff.png&quot;) 0 0 0 0 stretch stretch;
   border-style: outset;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="rightBlueButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>261</width>
      <height>271</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: #3C6291;
	border-image: url(&quot;:/icons/blueFlashOff.png&quot;) 0 0 0 0 stretch stretch;
   border-style: outset;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="score">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Terminal</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>501</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="youLostLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>521</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>67</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>YOU LOST</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="gameStatus">
    <property name="geometry">
     <rect>
      <x>426</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Terminal</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Watch!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>rightBlueButton</zorder>
   <zorder>leftRedButton</zorder>
   <zorder>playButton</zorder>
   <zorder>progressBar</zorder>
   <zorder>score</zorder>
   <zorder>youLostLabel</zorder>
   <zorder>gameStatus</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>542</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
